### PR TITLE
changed the alert after submition to message

### PR DESCRIPTION
**Pull request by naderakhgari**
Thank you for contributing! To help the review process, please provide the following:

### Proposal
changed the alert after submition to a message.

### Related
Provide links to any associated pull requests or issues. You can use `#123` to link to a PR or issue in this repository, or `user/repo#123` to link to other repositories.

### Checklist

- [ ] I have read the [code of conduct] and [contributing guide]
- [ ] I have made this pull request to the `master` branch
- [ ] I have run all of the automated validation using `npm run ship`
- [ ] I have added myself to the `"contributors"` list in the `package.json` (or do not want to)

[code of conduct]: https://github.com/textbook/starter-kit/blob/master/.github/CODE_OF_CONDUCT.md
[contributing guide]: https://github.com/textbook/starter-kit/blob/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: client/src/Components/Questions.js [hunk 1]
```diff
@@ -52,10 +52,8 @@ const Questions = (props) => {
 			body: JSON.stringify(answer),
 		});
 		event.target.reset;
-
-		if (currentQuestionIndex >= props.fetchedData.questions_id.length - 1) {
-			alert("You submited the form successfully!");
-		}
+		props.setIsSubmit(true);
```
Comment (cockochan) [inline client/src/Components/Questions.js:55]: I would suggest changing the name is to isSubmitted or startSubmit depending on what is happening.it confused me a bit when I first saw this name

File: client/src/App.css [hunk 2]
```diff
@@ -124,3 +124,48 @@
...
+  height: fit-content;
+  min-height: 200px;
+  min-width: 400px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  align-self: center;
+  z-index: 1;
+  opacity: 0.96;
```
Comment (cockochan) [inline client/src/App.css:146]: Oh wow. 0.04 opasity great attention to detail

File: client/src/App.css [hunk 3]
```diff
@@ -124,3 +124,48 @@
...
+  border-radius: 5px;
+}
+
+.close-container {
+  width: 95%;
+  background-color: rgba(4, 199, 4, 0.01);
```
Comment (cockochan) [inline client/src/App.css:152]: 0.01 opasity should be not too visible

File: client/src/Components/Students.js [hunk 4]
```diff
@@ -37,6 +41,8 @@ const Students = (props) => {
 					fetchedData={fetchedData}
```
Comment (cockochan) [inline client/src/Components/Students.js:41]: I try to understand if this lines are generated by GitHub to illustrate what is in the props? Probably it is just githubs version of the view illustrating what props student component takes

**Review by cockochan — APPROVED** [review]

File: client/src/Components/Questions.js (hunk 1 above)
Comment (naderakhgari) [reply client/src/Components/Questions.js:55]: It's right, I'll change it to isSubmitted

File: client/src/App.css (hunk 3 above)
Comment (naderakhgari) [reply client/src/App.css:152]: It's ok, we can change the css

File: client/src/Components/Students.js (hunk 4 above)
Comment (naderakhgari) [reply client/src/Components/Students.js:41]: I think yes

**Pull request merged**